src/main/scala/io/chrisdavenport/mules/Cache.scala

```diff
@@ -81,8 +84,8 @@ class Cache[F[_], K, V] private[Cache] (
 
 object Cache {
   // Value of Time In Nanoseconds
-  class TimeSpec private (
-    val nanos: Long
+  case class TimeSpec private (
```

Comment:
Does this expose the underlying unapply this way? Why did this shift happen?

Reply:
I can't really remember. Will change it back.